File: capabilities/oauth2/common/src/main/java/org/atricore/idbus/capabilities/oauth2/common/OAuth2AccessTokenEnvelope.java
package org.atricore.idbus.capabilities.oauth2.common;

/**
 * @author <a href=mailto:sgonzalez@atricore.org>Sebastian Gonzalez Oyuela</a>
 */
public class OAuth2AccessTokenEnvelope {

    private String signatureAlg;

    private String signatureValue;

    private String token;

    private boolean deflated;

    public OAuth2AccessTokenEnvelope(String signatureAlg, String signatureValue, String token, boolean deflated) {
        this.signatureAlg = signatureAlg;
        this.signatureValue = signatureValue;
        this.token = token;
        this.deflated = deflated;
    }

    public OAuth2AccessTokenEnvelope() {
    }

    public String getSignatureAlg() {
        return signatureAlg;
    }

    public String getSignatureValue() {
        return signatureValue;
    }

    public String getToken() {
        return token;
    }

    public boolean isDeflated() {
        return deflated;
    }

    public void setSignatureAlg(String signatureAlg) {
        this.signatureAlg = signatureAlg;
    }

    public void setSignatureValue(String signatureValue) {
        this.signatureValue = signatureValue;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public void setDeflated(boolean deflated) {
        this.deflated = deflated;
    }
}

File: capabilities/oauth2/common/common-dotnet/OAuth2Common/OAuth2AccessTokenEnvelope.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace OAuth2Common
{
    public class OAuth2AccessTokenEnvelope
    {

        private String encryptionAlg;

        private String signatureAlg;

        private String signatureValue;

        private String token;

        private Boolean deflated;

        public String EncryptionAlg
        {
            get { return encryptionAlg; }
            set { encryptionAlg = value; }
        }

        public String SignatureAlg
        {
            get { return signatureAlg; }
            set { signatureAlg = value; }
        }

        public String SignatureValue
        {
            get { return signatureValue; }
            set { signatureValue = value; }
        }

        public String Token
        {
            get { return token; }
            set { token = value; }
        }

        public Boolean Deflated
        {
            get { return deflated; }
            set { deflated = value; }
        }

     





    }
}
